src/validation.cpp

```diff
@@ -3040,7 +3047,7 @@ bool ContextualCheckBlock(const CBlock& block, CValidationState& state, const Co
     //   {0xaa, 0x21, 0xa9, 0xed}, and the following 32 bytes are SHA256^2(witness root, witness nonce). In case there are
     //   multiple, the last one is used.
     bool fHaveWitness = false;
-    if (VersionBitsState(pindexPrev, consensusParams, Consensus::DEPLOYMENT_SEGWIT, versionbitscache) == THRESHOLD_ACTIVE) {
+    if (VersionBitsState(pindexPrev, consensusParams, Consensus::DEPLOYMENT_SEGWIT2X, versionbitscache) == THRESHOLD_ACTIVE) {
```

Comment:
HF locking in on bit 1 activation would mean segwit could be activated while <80% support segwit2x.  I don't think that's consistent w/ Segwit2x's intent.  Better to make HF lock-in depend on bit 4, while making that HF contingent on segwit activating earlier - I believe this is how Sergio's Segwit2MB worked.  That way 1. Segwit2x doesn't break old BIP141 nodes, 2. HF can't happen w/o segwit, and 3. Neither activates unless Segwit2x reaches 80%.